jekyll/_sass/_sidebar.scss

```diff
@@ -142,46 +141,18 @@ $sidebar-spacing-base: 5px;
 }
 
 .sidebar-mobile-wrapper {
-  border-top: 1px solid $gray-200;
-  border-bottom: 1px solid $gray-200;
```

Comment:
This border-bottom created a visible end to the menu, without it this looks a bit odd with just the white slicing items in half. 
![Screen Shot 2021-04-26 at 11 43 32 AM](https://user-images.githubusercontent.com/5113432/116112394-852b4a00-a685-11eb-95bf-a3ac542c2bc9.png)

The border on `.mobile-sidebar`, which presumably replaces this, isn't visible until you've scrolled to the bottom of the menu.